firmware/application/ui_navigation.cpp

```diff
@@ -464,8 +464,11 @@ void NavigationView::focus() {
 /* ReceiversMenuView *****************************************************/
 
 ReceiversMenuView::ReceiversMenuView(NavigationView& nav) {
-	add_items({
-		//{ "..", 		ui::Color::light_grey(),&bitmap_icon_previous,	[&nav](){ nav.pop(); } },
+    if( portapack::persistent_memory::show_gui_return_icon() )
```

Comment:
Change looks good. Some general comments:
Could you: 
- check indentation
- squash commits?

Reply:
I've somewhat checked and in my editor it's globally okay. I have cleaned unnecessary tabs inside the func.
Note that most of the portopack files are not well indented at all. If I auto indent and commit, the whole file is touched except a few lines X-D

Reply:
I'll squash if I understand how to do it x-D

Reply:
You can squash when you merge. But not sure if you have access to merge, so we can squash and merge for you once ready

Reply:
I have just checked and I can not squash here.
It's okay for me :-)